packages/land/contracts/Land.sol

```diff
@@ -153,6 +164,18 @@ contract Land is LandBaseToken, OperatorFiltererUpgradeable {
         return royaltyManager;
     }
 
+    /// @notice Get the address of the owner
+    /// @return ownerAddress The address of the owner.
+    function owner() external view returns (address ownerAddress) {
+        return _owner;
+    }
```

Comment:
```
Contracts should also implement ERC165 for the ERC-173 interface.
```

Reply:
ERC165 cannot be added to Land as it'll cause storage conflicts

Reply:
I don't understand why, the function is here ```supportsInterface```, just have to add the interface id ?

Reply:
sorry, got confused. Will add and push